tests/components/fastdotcom/test_config_flow.py

```diff
@@ -70,5 +67,3 @@ async def test_import_flow_success(hass: HomeAssistant) -> None:
 
         assert result["type"] == FlowResultType.CREATE_ENTRY
         assert result["title"] == "Fast.com"
-        assert result["data"] == {}
-        assert result["options"] == {}
```

Comment:
Why are we removing?

Reply:
It's basically rubbish. It never set anything, so it's empty anyhow.

Reply:
It has been re-added.